src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/TimeoutExecutor.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.personrecord.service
+
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+
+object TimeoutExecutor {
+
+  fun runWithTimeout(timeout: Long, task: () -> Unit) = runBlocking {
+    withTimeout(timeout) {
```

Comment:
maybe just hard code this for now rather than wire the same value in everywhere and pass it through - would make this PR a lot smaller

Reply:
Can we get a test in for this somehow?
AIUI this will throw an exception so the message processing will fail.
:-)
Maybe set the test timeout to 3s or something and have an operation delay for 4s, then assert that the message being consumed ends up back on the queue?